cfme/tests/test_utilization_metrics.py

```diff
@@ -155,7 +204,7 @@ def test_raw_metric_vm_cpu(metrics_collection, appliance, provider):
         query = query_metric_db(appliance, provider, 'cpu_usagemhz_rate_average',
             vm_name)
         average_rate = attrgetter('cpu_usagemhz_rate_average')
-    elif provider.category == "cloud":
+    elif provider.category in "cloud container".split():
```

Comment:
If there are more words, that it makes sense to use split as it is shorter. The space is easier to type then `", "` and it also is IMHO more readable.

But as you wish.